packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -79,28 +79,28 @@ export const getStyles = memoizeFunction((
     },
 
     thumb: {
-      width: '.5em',
-      height: '.5em',
       borderRadius: '.5em',
       position: 'absolute',
       top: '.2em',
       transition: 'all 0.1s ease',
-      backgroundColor: thumbBackground,
+      borderColor: thumbBackground,
```

Comment:
Yup ! that my look better. Lets use that approach with proper comments that it transparent border is being added to support firefox HC mode to avoid anyone unknowingly removing that CSS.